components/fxa-client/ios/FxAClient/RustFxAccount.swift

```diff
@@ -252,6 +252,31 @@ open class RustFxAccount {
         }
     }
 
+    open func migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) throws -> Bool {
+        let json = try nullableRustCall { err in
+            fxa_migrate_from_session_token(self.raw, sessionToken, kSync, kXCS, 0, err)
```

Comment:
nit: I had to look up what this `0` was, could you please add a comment somewhere here to indicate that means we'll re-use the session token?